www/js/controllers/eventsController.js

```diff
@@ -0,0 +1,36 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2021 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
```

Comment:
This app is still GPL3. I wouldn't bother with adding file headers, it's not like anyone complained about the lack of them ;)